### ISSUE TITLE
Parametrization name should be parameter of I3SinglePandelFitter (Trac #1898)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1890]
In http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/lilliput/trunk/python/segments.py
it is not possible to pass to a segment (for example I3SinglePandelFitter but also the others) a parametrization service name or a minimizer name (that I can see, at least - if it's possible let me know). So, If I want to change parameters in the parametrization service, I need to give it the hard-coded name of "default_simpletrack" (which I have to find by looking into the code), otherwise it will be ignored. 

Migrated from https://code.icecube.wisc.edu/ticket/1898
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:55", 
    "description": "In http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/lilliput/trunk/python/segments.py\nit is not possible to pass to a segment (for example I3SinglePandelFitter but also the others) a parametrization service name or a minimizer name (that I can see, at least - if it's possible let me know). So, If I want to change parameters in the parametrization service, I need to give it the hard-coded name of \"default_simpletrack\" (which I have to find by looking into the code), otherwise it will be ignored. ", 
    "reporter": "delia.tosi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067175956889", 
    "component": "combo reconstruction", 
    "summary": "Parametrization name should be parameter of I3SinglePandelFitter", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-14T18:51:21", 
    "milestone": "", 
    "owner": "kkrings", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161014T18:55:17: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "kkrings"
* olivas changed component from "other" to "icerec"
* olivas changed type from "defect" to "enhancement"

**Comment by icecube-trac** [comment]
Trac update at 20161031T11:04:05: kkrings changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20161031T13:12:21: boersma commented:

> When I wrote the segment, the idea was that this was for performing the boring standard Pandel fit in just one line of your script. There are a few variations of the standard fit (like SPE, MPE), so that is why the segment has some configurable options, but other than that, everything is, well, standard. The segment is just a convenience wrapper around the single fitter module, so that you don't have to copy-paste the fitter module and its services and their configurations. If you want more configurability than the "standard" segment offers, then you can just add a single fitter module directly, with any combination of services that you like.
> 
> That was the idea. Of course ideas can change. Maybe there is a good use for a segment that is almost as flexible as the fitter module that's wrapped in it. I dunno.

**Comment by icecube-trac** [comment]
Trac update at 20161031T13:19:18: 
* kkrings changed milestone from "" to "November 2016"
* kkrings commented:

> I think it makes sense to make this segment more flexible so that you can replace for example only one part of the standard configuration like the parametrization.

**Comment by icecube-trac** [comment]
Trac update at 20170120T14:56:08: 
* kkrings commented:

> In changeset 152830:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="152830"
> Support external minimizer and parametrizations in segments; closes #1890.
> ```


* kkrings changed resolution from "" to "fixed"
* kkrings changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:55: 
* olivas commented:

> Milestone November 2016 deleted


* olivas changed milestone from "November 2016" to ""
